CONTRIBUTING.md

```diff
@@ -169,6 +169,21 @@ To start n8n with tunnel:
 ./packages/cli/bin/n8n start --tunnel
 ```
 
+## First run
+
+The first time you run n8n locally you will be prompted to create an owner user.
+The owner user credentials are stored in the `n8n` database and are needed in subsequent runs to access the n8n UI.
+
+The location of the database is under $HOME/.n8n/database.sqlite (unless explicitly overridden).
+If for any reason the credentials are lost before setting up password recovery, you can either remove the database file or run the following SQL command to reset the owner user:
+
+```
+update settings SET value='false' where key='userManagement.isInstanceOwnerSetUp';
```

Comment:
We would recommend using the server command over directly editing the database: https://docs.n8n.io/hosting/cli-commands/#user-management